source/style-guide/markup/directives/code-blocks.txt

```diff
@@ -2,20 +2,444 @@
 Code Examples
 =============
 
-As needed, use the appropriate language option for code blocks directives
-including the following directives:
+Snooty supports three directives for including code samples in a docs page:
 
 - ``code-block``
 - ``io-code-block``
 - ``literalinclude``
 
-These directives feature syntax highlighting which makes the source code 
-contained in them easier to interpret. The following example shows how to 
-specify Python language syntax highlighting in a ``code-block`` directive:
+These directives feature a monospace font and syntax highlighting that makes the
+source code contained in them easier to interpret.
+
+Code Blocks
+-----------
+
+The ``code-block`` directive formats a code sample written beneath the
+directive.
+
+Syntax
+~~~~~~
 
 .. code-block::
 
+   .. code-block:: <language>
+      :copyable: <boolean>
+      :emphasize-lines: <line numbers>
+      :linenos:
+      :caption: <caption text>
+      :source: <url>
+
+      <code sample>
+
+Options
+~~~~~~~
+
+.. list-table::
+
+   * - ``<language>``
+     - The language to use for syntax highlighting. 
+       
+       For a complete list of supported languages, see the
+       `leafygreen-ui GitHub repository. <https://github.com/mongodb/leafygreen-ui/tree/main/packages/code#properties>`__
+
+       **Default:** None
+ 
+   * - ``:copyable: <boolean>``
+     -  Whether to show the Copy button for the code sample.
```

Comment:
I think it's because it's a div, which lacks an alt text tag. However, I reached out to Design Systems to ask whether they could add text when you hover over it which could be better than nothing.

Reply:
Following up for @corryroot 
Here's the solution the design systems team decided upon: https://jira.mongodb.org/browse/LG-3874

Reply:
@ccho-mongodb, awesome. Thanks!